websocket/polygon.go

```diff
@@ -128,7 +128,18 @@ func (c *Client) connect() error {
 	return nil
 }
 
+func (c *Client) checkPrefix(topic Topic) {
+	if _, prefixExists := c.subscriptions[topic.prefix()]; !prefixExists {
+		c.subscriptions[topic.prefix()] = make(set)
+	}
+}
+
 func (c *Client) Subscribe(topic Topic, tickers ...string) error {
```

Comment:
i don't think this supports the `Subscribe(StocksSecAggs)` case anymore. if no tickers are passed, we should assume the user means `Subscribe(StocksSecAggs, "*")`. probably worth adding this to the live test as well (or modifying the part that you added). Unsubscribe should do the same thing